### ISSUE TITLE
Tutorial seemingly inconsistent with main documentation

### ISSUE THREAD

**Issue by escalonn** [aspnet/Docs#7200]
This tutorial places the layout page, and some partials, in _Pages/Shared_ folder, while the [main documentation](https://docs.microsoft.com/en-us/aspnet/core/razor-pages/?view=aspnetcore-2.1&tabs=visual-studio#using-layouts-partials-templates-and-tag-helpers-with-razor-pages) for Razor Pages says this:
"We recommend you *not* put the layout file in the _Views/Shared_ folder. _Views/Shared_ is an MVC views pattern. Razor Pages are meant to rely on folder hierarchy, not path conventions."
Clarification in one place or the other is necessary.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a25bf940-8977-fec4-461b-542a9f550fc8
* Version Independent ID: 5492c641-af15-64b8-beaa-ddf511be73e6
* Content: [Get started with Razor Pages in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/tutorials/razor-pages/razor-pages-start?view=aspnetcore-2.1)
* Content Source: [aspnetcore/tutorials/razor-pages/razor-pages-start.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/tutorials/razor-pages/razor-pages-start.md)
* Product: **aspnet-core**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

**Comment by AndySmiffy** [comment]
So out of interest which is correct?

**Comment by guardrex** [comment]
* ASP.NET Core 2.0 Razor Pages: **_Pages_**
* ASP.NET Core 2.1 or later Razor Pages: **_Pages/Shared_**
* Mixed app of Razor Pages + MVC that uses a common layout page: We're waiting to hear back from engineering on what they suggest. See: https://github.com/aspnet/Docs/issues/7336#issuecomment-401577370

**Comment by Rick-Anderson** [comment]
>We recommend you not put the layout file in the Views/Shared folder. Views/Shared is an MVC views pattern. Razor Pages are meant to rely on folder hierarchy, not path conventions.

That's all correct. You don't put Razor Pages layout files in the MVC Views/Shared folder

**Comment by Rick-Anderson** [comment]
>hile the main documentation for Razor Pages says to put the layout page directly in Pages folder,

Where does it say that in the 2.1 version?

**Comment by elias-alea** [comment]
Just to add on to this, I created a 2.1 app and the _Layout.cshtml is under Pages/Shared. 
Also, the _viewStart.cshtml looks like
@{
    Layout = "_Layout";
}
will it still work while _ViewImports.cshtml declares the namespace as @namespace RazorPagesMovie.Pages?
